server/config/seed.js

```diff
@@ -36,38 +44,46 @@ const insertSeed = async () => {
 
 insertSeed();
 
-(async () => {
-  let result;
-  const params = [
-    'martinsUsername',
-    'martins@gmail.com',
-    'martins',
-    'obayomi',
-    'martinsPw'];
-  try {
-    result = await pool.query(`INSERT INTO users 
-    (username, email, firstname, lastname, password)
-      VALUES ($1, $2, $3, $4, $5)`, params);
-    return result;
-  } catch (error) {
-    return error;
-  }
-})();
+// (async () => {
```

Comment:
Rockstar Tolu! Awesome job man. This is impressive on many levels. The thoughtfulness you put into it is admirable.

Is there any reason to keep this code that has been commented out?

Reply:
Thanks for pointing that out @LukasChiama I actually forgot to delete that, but I've done that now